src/EntityFramework6.Npgsql/EntityFramework6.Npgsql.nuspec

```diff
@@ -16,7 +16,7 @@
     <language>en-US</language>
     <tags>npgsql postgresql postgres data database entity framework ef orm</tags>
     <dependencies>
-      <dependency id="Npgsql" version="3.1.0" />
+      <dependency id="npgsql" version="3.1.0" />
       <dependency id="EntityFramework" version="6.0.0" />
```

Comment:
I think there is a minor chance that roji might want to build nuget packages for earlier releases with the new package name, or someone might have their own forks of earlier releases. I don't think it would add any value to unnecessarily restrict it to 3.1.5 and higher when it is not absolutely needed.

I am using a fork of this project with this fix integrated in a .NET core project and it is resolving and functioning correctly as-is in this pull request, so I know this works. I think I will defer to @roji on this. If he thinks it should start at 3.1.5, then I'll change it.